osu.Game.Tests/Beatmaps/Formats/LegacyBeatmapDecoderTest.cs

```diff
@@ -482,5 +482,23 @@ public void TestDecodeHitObjectNullAdditionBank()
                 Assert.AreEqual(hitObjects[0].Samples[0].Bank, hitObjects[0].Samples[1].Bank);
             }
         }
+
+        [Test]
+        public void TestDecodeInvalidEvents()
```

Comment:
this test is very nondescript. someone looking at this is going to WTF. there's no asserts on events of any kind, so what is it even doing?

Reply:
You can kind of argue the same for the rest of the tests in this file though? I agree the name doesn't really help much for the cause though. Would it be alright if I renamed + xmldocd this test?